app/lib/frontend/dom/dom.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+
+final _attributeEscape = HtmlEscape(HtmlEscapeMode.attribute);
+
+/// The DOM context to use while constructing nodes.
+///
+/// Override this in browser.
+DomContext dom = StringDomContext();
```

Comment:
nit...
```suggestion
DomContext html = StringDomContext();
```

Then things are written as `html.div` and `html.fragment`